src/core/Track.cpp

```diff
@@ -688,9 +688,7 @@ void TrackContentObjectView::paintTextLabel(QString const & text, QPainter & pai
 void TrackContentObjectView::mousePressEvent( QMouseEvent * me )
 {
 	setInitialMousePos( me->pos() );
-
-	BBTrackContainer * isbbEditor = dynamic_cast<BBTrackContainer*>( getTrackContentObject()->getTrack()->trackContainer() );
-	if( !isbbEditor && me->button() == Qt::LeftButton )
+	if( me->button() == Qt::LeftButton && fixedTCOs() == false )
```

Comment:
You could just to `!FixedTCOs()` since it's a boolean

Reply:
I wanted to be consist with the rest of the code. But if there are no objections I do it for the whole If statement.

Reply:
`!FixedTCOs()` because I literally read it "If it's not a Fixed TCO"